docs/excel/add-properties-to-basic-cell-values.md

```diff
@@ -0,0 +1,398 @@
+---
+title: Add properties to basic cell values
+description: Add properties to basic cell values.
+ms.topic: how-to #Required; leave this attribute/value as-is
+ms.date: 03/17/2025
+ms.localizationpriority: medium
+---
+
+# Add properties to basic cell values
+
+Add properties to basic cell values in Excel to associate additional information with the values. Similar to entity values, you can add properties to the **string**, **double**, and **Boolean** basic types. Each property is a key/value pair. The following example shows the number 104.67 (double) that represents a bill with added fields named **Drinks**, **Food**, **Tax**, and **Tip**.
+
+:::image type="content" source="../images/data-type-basic-fields.png" alt-text="Screen shot of the drinks, food, tax, and tip fields shown for the selected cell value.":::
+
+If the user chooses to show the data type card, they will see the values for the fields.
+
+:::image type="content" source="../images/data-type-basic-data-type-card.png" alt-text="Data type card showing values for drinks, food, tax, and tip properties":::
+
+Cell value properties can also be used in formulas.
+
+:::image type="content" source="../images/data-type-basic-dot-syntax.png" alt-text="Show user typing "a1." and Excel showing a menu with drinks, food, tax, and tip options.":::
```

Comment:
I think this will fix the publishing error.
```suggestion
:::image type="content" source="../images/data-type-basic-dot-syntax.png" alt-text="Show user typing 'a1.' and Excel showing a menu with drinks, food, tax, and tip options.":::
```